src/de/2024-04/06/teacher-questions.md

```diff
@@ -3,15 +3,15 @@ title:  Gesprächsfragen SabbatschulWiki
 date:   08/11/2024
 ---
 
-#### Startfrage(n)
+#### **Startfrage(n)**
```

Comment:
@toak There is no point in bolding the headings, they are always rendered with bold font

Reply:
will always be removed